### PR TITLE
Fix: add configuration api reference

### PR DESCRIPTION
**Pull request by LaurenceLiZhixin**
Signed-off-by: LaurenceLiZhixin <382673304@qq.com>
This is for issue https://github.com/dapr/docs/issues/2493

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Issue reference

The component docs would be updated after we reach an agreement of issue https://github.com/dapr/dapr/issues/4546

### CODE REVIEW

**Comment by msfussell** [comment]
@LaurenceLiZhixin - Thanks for getting this started. I have left a comment in the reference doc. However you have included a large number of other files that you will need to remove from the PR. I suggest starting new PR.

As well as the reference doc, these two articles need to be updated, especially the code examples showing HTTP. I can work with you here. 
https://docs.dapr.io/developing-applications/building-blocks/configuration/configuration-api-overview/
https://docs.dapr.io/developing-applications/building-blocks/configuration/howto-manage-configuration/

File: daprdocs/content/en/reference/api/configuration_api.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+#### URL Parameters
+
+Parameter | Description
+--------- | -----------
+`daprPort` | The Dapr port
+`storename` | The `metadata.name` field in the user-configured `configuration.yaml` component file. Refer to the [Dapr configuration store component file structure](#component-file) mentioned above.
+
+The optional request, like configuration key and metadata are passed via URL query parameters. For example:
+```
+GET http://localhost:3500/v1.0/configuration/myConfigurationStore?key=myConfigKey&metadata.contentType=application/json
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/configuration_api.md]: I would have expected this to have a key value in the URL and not in the query parameters, 
to be consistent with the state management GET API. Like this?

GET http://localhost:3500/v1.0/configuration/myConfigurationStore/myConfigKey

File: daprdocs/content/en/reference/api/configuration_api.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+| `spec/metadata` | An open key value pair metadata that allows a binding to define connection properties. |
+
+## Get Configuration
+
+This endpoint lets you get configuration from store.
+
+### HTTP Request
+
+```
+POST http://localhost:<daprPort>/v1.0/configuration/<storename>
```
Comment (msfussell) [inline daprdocs/content/en/reference/api/configuration_api.md]: Should this not be 
```suggestion
GET http://localhost:<daprPort>/v1.0/configuration/<storename>/<key>
```

Comment (LaurenceLiZhixin) [reply daprdocs/content/en/reference/api/configuration_api.md]: Yeah, it should be GEY method. I 'll fix it.

**Comment by LaurenceLiZhixin** [comment]
> @LaurenceLiZhixin - Thanks for getting this started. I have left a comment in the reference doc. However you have included a large number of other files that you will need to remove from the PR. I suggest starting new PR.
> 
> As well as the reference doc, these two articles need to be updated, especially the code examples showing HTTP. I can work with you here. https://docs.dapr.io/developing-applications/building-blocks/configuration/configuration-api-overview/ https://docs.dapr.io/developing-applications/building-blocks/configuration/howto-manage-configuration/

OK, I'll re-open a new pr, and fix problem you mentioned.

**Comment by msfussell** [comment]
@LaurenceLiZhixin - Would like to get this updated soon, given we are now into the endgame. Cannot release without this. Thanks

**Comment by yaron2** [comment]
This PR is targeting the wrong branch. @LaurenceLiZhixin I'm closing this PR and will submit an updated PR for 1.8.

**Pull request closed without merging**